File: price_fetcher.py
import pg8000
import requests

api = "https://api.pro.coinbase.com"

try:
	connect = pg8000.connect(user="btc", password="Bitcoin")
	cursor = connect.cursor()

	time_request = requests.get(api+"/time")
	time_data = time_request.json()
	formatted_time = time_data["iso"][0:10] + " " + time_data["iso"][11:19]

	price_request = requests.get(api+"/products")
	price_data = price_request.json()
	i = 0
	while (price_data[i]["id"] != "BTC-USD"):
		i=i+1

	# Format is timestamp, numeric(7,2)
	# ('YYYY-MM-DD HH:MM:SS', xxxxx.yy)
	cursor.execute("INSERT INTO btc_price ()")

except pg8000.DatabaseError:
	

finally:
	

File: PriceFetcher.java
import java.sql.*;
import java.net.*;
import java.io.*;

public class PriceFetcher
{
	String url = "jdbc:postgresql:btc";
	
	public double get_price()
	{
		try
		{
			URLConnection connection = new URL("https://api.pro.coinbase.com/products/btc-usd/ticker").openConnection();
			BufferedReader input = new BufferedReader(new InputStreamReader(connection.getInputStream()));
			return(0.0);
		}
		catch (Exception e)
		{
			e.printStackTrace();
			return(-1.0);
		}
	}
	
	public boolean table_insert()
	{
		// Format is timestamp, numeric(7,2)
		// ("YYYY-MM-DD HH:MM:SS", xxxxx.yy)
		try
		{
			Connection db = DriverManager.getConnection(url, "btc", "Bitcoin");
			return(true);
		}
		catch (Exception e)
		{
			e.printStackTrace();
			return(false);
		}
	}
	
	public static void main(String[] args)
	{
		PriceFetcher fetcher = new PriceFetcher();
		try
		{
			
		}
		catch (Exception e)
		{
			e.printStackTrace();
		}
	}
}
